.github/workflows/gpu.yml

```diff
@@ -88,3 +88,7 @@ jobs:
       - name: Run examples
         working-directory: ${{github.workspace}}/build
         run: ctest --output-on-failure --test-dir examples -C ${{env.BUILD_TYPE}}
+
+      - name: Run benchmarks
+        working-directory: ${{github.workspace}}/build
+        run: ctest --output-on-failure --test-dir benchmarks -C ${{env.BUILD_TYPE}} --exclude-regex umf-bench-multithreaded
```

Comment:
I believe the directory is called `benchmark` (singular)

Reply:
right, I made a typo.

Fixed